### ISSUE TITLE
empty ame jars are visible above AI static plane

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#30211]
(WEB REPORT BY: gurfan REMOTE: 206.221.180.138:7777)
# Revision
0d84d10bf7318d606f78e328d8ffa6a693457846

# Description
the title

# Steps to Reproduce
be ai, see empty ame jar over static

# What you Expected
to not see ame jar over static

# What Actually Happened
i shidded and farted

**Comment by boy2mantwicethefam** [comment]
Feature

**Comment by DeityLink** [comment]
Ok I have no idea about that one, neither the jar's own code nor the control unit edit planes in any way. The problem might not be specific to antimatter jars. Need more testing. @gurfan

**Comment by gurfan** [comment]
yeah I had a feeling that this was some one-off thing, I'll try to test it more as well
